tests/playwright/src/specs/compose-onboarding-smoke.spec.ts

```diff
@@ -140,7 +158,30 @@ test.describe.serial('Compose onboarding workflow verification', { tag: '@smoke'
     const cliToolsPage = await settingsBar.openTabPage(CLIToolsPage);
     const composeRow = cliToolsPage.toolsTable.getByLabel(RESOURCE_NAME);
     const composeVersionInfo = composeRow.getByLabel('cli-version');
-    await playExpect(composeVersionInfo).toHaveText('docker-compose ' + composeVersion);
+    await playExpect(composeVersionInfo).toContainText('docker-compose ' + composeVersion);
+  });
+
+  test('Check Podman Desktop autorefresh when using podman compose up', async ({ navigationBar }) => {
+    test.setTimeout(300_000);
+
+    const composeFilePath = path.resolve(__dirname, '..', '..', 'resources', `compose.yaml`);
```

Comment:
I think we could divide the test into two. first would be verification of `podman compose up` - we could say we are veryfing that installed binary actually works if command execution ends up without error. And then we check the compose containers and that they are marked as `compose`.